### PR TITLE
ebnf trace

### LINKED ISSUE #16995: sql-statements/sql-statement-trace.md

### PR DESCRIPTION
**Pull request by dveeden**
### What is changed, added or deleted? (Required)

Closes #16995

Also add `FORMAT='log'`

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions](https://github.com/pingcap/docs/blob/master/CONTRIBUTING.md#guideline-for-choosing-the-affected-versions).

- [x] master (the latest development version)
- [x] v8.1 (TiDB 8.1 versions)
- [x] v8.0 (TiDB 8.0 versions)
- [ ] v7.6 (TiDB 7.6 versions)
- [ ] v7.5 (TiDB 7.5 versions)
- [ ] v7.4 (TiDB 7.4 versions)
- [ ] v7.1 (TiDB 7.1 versions)
- [ ] v6.5 (TiDB 6.5 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

**Review by lilin90 — APPROVED** [review]
/lgtm

**Review by kennytm — APPROVED** [review]

**Comment by dveeden** [comment]
We could use tabs for the different formats, just like we do for EXPLAIN

File: sql-statements/sql-statement-trace.md [hunk 1]
```diff
@@ -10,20 +10,26 @@ The `TRACE` statement provides detailed information about query execution. It is
...
+TracableStmt ::=
+    ( SelectStmt | DeleteFromStmt | UpdateStmt | InsertIntoStmt | ReplaceIntoStmt | UnionStmt | LoadDataStmt | BeginTransactionStmt | CommitStmt | RollbackStmt | SetStmt )
+```
 
-![TraceableStmt](/media/sqlgram/TraceableStmt.png)
+| Format | Description                        |
+|--------|------------------------------------|
+| row    | Output in a tree format            |
+| log    | Log based output                   |
+| json   | Structured output in JSON format   |
```
Comment (lilin90) [inline sql-statements/sql-statement-trace.md:25]: To keep the order consistent with that in the examples section.
```suggestion
| json   | Structured output in JSON format   |
| log    | Log based output                   |
```

**Comment by lilin90** [comment]
> We could use tabs for the different formats, just like we do for EXPLAIN

Yeah, that's a choice. The "Examples" section of https://docs.pingcap.com/tidb/stable/sql-statement-explain is relatively long. For the document in this PR, I think it's also clear to keep it as it is.

**Comment by lilin90** [comment]
> We could use tabs for the different formats, just like we do for EXPLAIN

Yeah, that's a choice. The "Examples" section of https://docs.pingcap.com/tidb/stable/sql-statement-explain is relatively long. For the document in this PR, I think it's also clear to keep it as it is.

**Review by lilin90 — COMMENTED** [review]
/approve

**Pull request merged**